### ISSUE TITLE
I really enjoy your work. Can this project generate panoramic images directly from pictures? How much VRAM does it require and what type of open source license is it

### ISSUE THREAD

**Issue by libai-lab** [zxcvfd13502/PanoFree#1]
I really enjoy your work. Can this project generate panoramic images directly from pictures? How much VRAM does it require and what type of open source license is it

**Comment by zxcvfd13502** [comment]
Yes, PanoFree can also be used to inpaint a panoramic image from give several views, as it's based on iterative generation pipeline which naturally support this. We list the required GPU memory in our paper, and this could be further decreased with other efficient deployment techniques of diffusion models.
